cs/src/core/Index/Common/RecordInfo.cs

```diff
@@ -228,26 +238,33 @@ public bool Dirty
             }
         }
 
-        public bool Invalid
```

Comment:
What is Filler?

Reply:
We will use Filler to track unused space in the main log, which might be present if we resized a value from say 100 down to 60. This will allow us to in-place update the value back from 60 to say 80.

Reply:
Specifically, the plan is: if Filler = true, then the 8 bytes following Value (or Key in case of Tombstone) will denote the size of the filler space, i.e. the space in bytes before the start of the next RecordInfo on that page.